A2/plugins/util.py

```diff
@@ -17,3 +19,23 @@ def ping_command(self, event):
         user_bot_latency = (bot_ping - user_ping).total_seconds() * 1000.0
 
         bot.edit('Latency of you to bot: ~{:.2f}ms'.format(user_bot_latency))
+
+    @Plugin.command('choose', '<options:str...>') 
+    def choose_command(self, event, options):
+        """= choose =
+        Randomly choose between two or more options
+        usage    :: $choose <options...>
+        aliases  :: None
+        category :: Utilities
+        == Arguments
+        options  :: A list of options from which one will be chosen
+        == Examples
+        $choose windows\macOS\linux 'I choose: macOS'
+        $choose boil em\mash em\stick em in a stew 'I choose: boil em'
+        $choose a\ \c 'I choose: c'
+        """
+        options = options.split('\\')
+        options = [i for i in options if i.strip() != '']
```

Comment:
Change `i` to `option` and you should be returning the stripped string to the list and your condition should be checking if the string is empty. Something like this: `[option.strip() for option in options if option.strip()]`. I don't think it would even be *too* cumbersome if you put the `options.split('\\')` part inside the list comprehension.

Lastly you need to check if your `options` list is empty or only has one item *before* you pick a random item from that list. So do a check to see that more than 1 item exists in `options` after the list comprehension, if there isn't, `return`, if there is, then you're free to pick a random item and display it.